javascript/javascript1/week3/Homework/Exercise5.js

```diff
@@ -14,35 +15,57 @@ function addActivity(date, activity, duration) {
   activities.push(newActivity);
 }
 
-addActivity("23/7-18", "YouTube", 30);
-console.log(activities);
-
-// Show my status
-// Create a function called showStatus. This function should use the activities variable and return a string saying the following: "You have added 3 activities. They amount to 78 min. of usage"
-
 function calculateTotalDuration(total, activity) {
   return total + activity.duration;
 }
 
-function showStatus(activities) {
+function showStatus(date = new Date().toLocaleDateString()) {
+  const filteredActivities = activities.filter(
+    (activity) => activity.date === date
+  );
+
+  if (filteredActivities.length === 0) {
+    return `No activities found for ${date}`;
+  }
+
+  let totalActivities = filteredActivities.length;
```

Comment:
I would declare this variable further up before the if statement and refer to it there when checking it is `=== 0` and make it a `const` as you are not re-assigning it.